src/preprocessing/passes/ackermann.cpp

```diff
@@ -187,9 +188,110 @@ void collectFunctionsAndLemmas(FunctionToArgsMap& fun_to_args,
 
 /* -------------------------------------------------------------------------- */
 
+/* Given a minimum capacity for an uninterpreted sort, return the size of the
+ * new BV type */
+size_t getBVSkolemSize(size_t capacity)
+{
+  return static_cast<size_t>(log2(capacity)) + 1;
+}
+
+/* Given the lowest capacity requirements for each uninterpreted sort, assign
+ * a sufficient bit-vector size. Get the converting map */
```

Comment:
We are no longer assigning a sufficient bv size, right? It is just the lowest capacity itself I think.

Reply:
it is sufficient, because 2^size may greater than the lowest capacity